### DISCUSSION TITLE
Is it possible to redirect from subdomain to external page?

### DISCUSSION THREAD

**Discussion by imonzalvo** [vercel/community#2679] [Help]
### Summary

Hey there! I' having some troubles working with redirects. I want to redirect all my traffic entering from `test.domain.io` to `app.domain.io`

I'm using a project with just a `vercel.json` file to redirect. And this always thros 404.

```
{
    "rewrites": [
        {
            "source": "/:path*",
            "has": [
                {
                    "type": "host",
                    "value": "test.domain.io"
                }
            ],
            "destination": "app.domain.io/:path*"
        }
    ]
}
```

### Steps to Reproduce

Recreate projects with the config mentioned

**Comment by amyegan** [comment]
If you're [rewriting](https://vercel.com/docs/concepts/projects/project-configuration#rewrites) to an external site, you need to include the protocol like `https://example.com/:path*`. If both domains are on the same project, you can more easily configure a redirect in the domain settings

![image](https://github.com/vercel/community/assets/8247105/3d170915-13cf-4352-bc21-012955d4dfbf)

**Reply by monzalvoignacio** [reply]
Actually, those domains aren't on the same project. But also tried adding https:// and didn't work.
